README.md

```diff
@@ -1,6 +1,6 @@
-# Coreto Ledger SDK
+# Coreto DRT SDK
 
-An SDK used to interact with the Coreto Trust and Performance Ledger
+An Laravel SDK used to interact with the Coreto DRS
```

Comment:
Is this only Laravel specific, or can it be used in other PHP frameworks as well?

Reply:
We are using `Illuminate` for HTTP calls, which is a part of Laravel, so yes, it is Laravel specific.